portfolio/src/main/webapp/script.js

```diff
@@ -82,6 +89,35 @@ function renderListComments({firstName, lastName, commentText, date}) {
   document.getElementById('comments').appendChild(content);
 }
 
+function createMap() {
+  const map1 = new google.maps.Map(document.getElementById('map'), {
+    center: {lat: 19.741755, lng: -155.844437},
+    zoom: 2,
+    disableDefaultUI: true,
+    styles: MAP_STYLE
+    });
+  showInfoWindowAndMarker(map1, PLACES);
+}
+
+/** 
+ * Creates a window with some context about a place at given marker on the map.
+ * 
+ * @param {google.maps.Map} map map where to show the window
+ * @param {object[]} PLACES array which contains object with place location and info
+ */
+function showInfoWindowAndMarker(map, PLACES) {
+  PLACES.forEach((place) => {
+
+    const marker = new google.maps.Marker({
+      position: place.latLng,
+      map: map,
```

Comment:
if the field and variable have the same name, the field can be omitted:

```
const marker = new google.maps.Marker({
  position: place.coordinate,
  map,
  title: place.contentText
});
```

Reply:
Thanks for the context. Changes made.